lms/static/sass/shared/_header.scss

```diff
@@ -377,6 +388,19 @@ header.global-new {
 
   .nav-global {
     @include float(left);
+    .tab-nav-item{
+      .tab-nav-link{
+        height: 43px;
+        font-size: em(16);
+        color: $gray;
+        &:hover{
```

Comment:
By the pattern library tabs do you mean the ones for the pattern library doc site (ux.edx.org)?